src/wren/FrameBuffer.cpp

```diff
@@ -159,22 +159,20 @@ namespace wren {
     glstate::bindReadFrameBuffer(currentReadFrameBuffer);
   }
 
-  void FrameBuffer::copyContents(size_t index, void *data) {
+  void FrameBuffer::copyContents(size_t index, void *data, int offset, int size) {
     assert(index < mOutputDrawBuffers.size());
     assert(mOutputDrawBuffers[index].mGlNamePbo);
+    assert(src + size < mWidth * mHeight);
```

Comment:
I am surprised this compiled...
```suggestion
    assert(offset + size < mWidth * mHeight);
```

Reply:
It seems that even putting assert(0) doesn't prevent the program to continue..